### PR TITLE
[WIP] Spanish translation

### PR DESCRIPTION
**Pull request by stsewd**
Hi, awesome project! I'll be translating the rest of files during this week :)

## Translated files
- [x] context.cson
- [x] menu_darwin.cson
- [x] menu_linux.cson
- [x] menu_win32.cson
- [x] settings.cson

### CODE REVIEW

**Comment by liuderchi** [comment]
@stsewd Thank you so much for contributing :100: 

Really looking forward to Atom in Spanish

**Comment by stsewd** [comment]
@liuderchi Hi, I just finish the Spanish translation, feel free to merge the pull request, if there is something wrong, please let me know, I will try to fix it the more sooner possible.

_Notes for future Spanish translators/Nota para futuros traductores: He dejado algunas palabras en inglés, dado que no tienen mucho sentido si son traducidas, y no encontré una traducción más "natural". Sugerencias son bienvenidas._

**Comment by liuderchi** [comment]
LGTM and ready to be merged.

Please keep updating Atom and watching for new UI changes.
If found new Item require translation, please send issues.

Thanks so much @stsewd for your contribution :100:

File: def/es/settings.cson [hunk 1]
```diff
@@ -11,378 +11,405 @@ Settings:
...
-      {
-        _id: 'editor-settings-note'
-        html: 'These settings are related to text editing. Some of these
-               can be overriden on a per-language basis. Check language
-               settings by clicking its package card in the
-               <a class="link packages-open">Packages list</a>.'
+        html: 'Estos son los ajustes de Atom, los cuales no afectan
+               el comportamiento relacionado a editar texto. Paquetes
+               individuales pueden tener sus propios ajustes en la
+               <a class="link packages-open">lista de Paquetes</a>.'
```
Comment (liuderchi) [inline def/es/settings.cson]: looks like we missed translation of section `'editor-settings-note'`

Comment (stsewd) [inline def/es/settings.cson]: Sorry that I skipped that one, I'll translate it later.

**Pull request merged**